src/items/equipment.js

```diff
@@ -45,18 +47,22 @@ export default class Equipment extends Item {
 
     equip() {
         if (!this.equipped) {
-            console.log("Equipping", this.name);
+            if (this.checkRequiredLevel()) {
+                console.log("Equipping", this.name);
 
-            // Remove from inventory if it was there
-            if (this.index >= 0) {
-                this.scene.characterData.inventory[this.index] = {};
-                this.scene.inventory.obj.inventory[this.index] = {};
-                this.index = -1;
-            }
+                // Remove from inventory if it was there
+                if (this.index >= 0) {
+                    this.scene.characterData.inventory[this.index] = {};
+                    this.scene.inventory.obj.inventory[this.index] = {};
+                    this.index = -1;
+                }
 
-            this.actions[0] = { text: "Unequip", func: "unequip" };
-            this.equipped = true;
-            this.scene.equipment.obj.equipItem(this);
+                this.actions[0] = { text: "Unequip", func: "unequip" };
+                this.equipped = true;
+                this.scene.equipment.obj.equipItem(this);
+            } else {
+                console.log("Not high enough level to equip that.");
```

Comment:
Might want to put something on the trello board to make a logging level that prints to the chat window. This is something that would go there